### PR TITLE
Completed javascript-loops exercise - Malcolm Abbott

### PR DESCRIPTION
**Pull request by Malcolm-Abbott**
javascript-loops-completed

### CODE REVIEW

File: javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+    let numbers = [];
```
Comment (shawnkost) [inline javascript-loops/main.js]: This should be using `const`

File: javascript-loops/main.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    let currentNumber = 1;
+    while (currentNumber < 11) {
+        numbers.push(currentNumber);
+        currentNumber++;
+    } return numbers;
+}
+
+function getEvenNumbersToTwenty() {
+    let evenNumbers = [];
```
Comment (shawnkost) [inline javascript-loops/main.js]: This should be using `const`

File: javascript-loops/main.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+function logEachCharacter(string) {
+    for (i = 0; i < string.length; i++) {
+        console.log(string[i]);
+    }
+}
+
+function doubleAll(numbers) {
+    let doubled = [];
```
Comment (shawnkost) [inline javascript-loops/main.js]: This should be using `const`

File: javascript-loops/main.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    for (i = 0; i < numbers.length; i++) {
+        doubled.push(numbers[i] * 2);
+    } return doubled;
+}
+
+function getKeys(object) {
+    let keys = [];
```
Comment (shawnkost) [inline javascript-loops/main.js]: This should be using `const`

File: javascript-loops/main.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    for (let key in object) {
```
Comment (shawnkost) [inline javascript-loops/main.js]: This should be using `const`

File: javascript-loops/main.js [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+        keys.push(key);
+    } return keys;
+}
+
+function getValues(object) {
+    let values = [];
+    for (let key in object) {
+        let value = object[key];
```
Comment (shawnkost) [inline javascript-loops/main.js]: These should be using `const`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some tweaks here Malcolm! When we declare a variable for arrays & objects, we should still be using `const` rather than `let`. Even though we add & remove items from them, we never actually reassign the whole variable to something else, so it should still be `const`.

**Comment by Malcolm-Abbott** [comment]
recommitted and pushed with the fixes you requested, thank you very much!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**